README.md

```diff
@@ -25,6 +25,14 @@ There are 2 types of navigators available:
 
 See how to get you instance for [StackNavigation](src/StackNavigation.Abstractions/StackNavigation.md#getting-started) or [SectionsNavigation](src/SectionsNavigation.Abstractions/SectionsNavigation.md#getting-started). Note that your code should always use the interface in order to be easily reused for integration tests.
 
+### Navigation Types 
+
+#### Page Navigation
+![Page Navigation](docs/images/PageNavigation.png)
```

Comment:
Those images should be under the bullet points in the "Choose your Navigator"  section.
However, no image matches the requirement for StackNavigator.